packages/tpetra/core/src/Tpetra_Details_CrsPadding.hpp

```diff
@@ -28,20 +28,13 @@ namespace Tpetra {
       };
 
     public:
-      struct create_from_sames_and_permutes_tag {};
-      static constexpr create_from_sames_and_permutes_tag
-        create_from_sames_and_permutes {};
-      CrsPadding(create_from_sames_and_permutes_tag /* tag */,
-                 const int myRank,
+      CrsPadding(const int myRank,
```

Comment:
Thanks @jhux2 !  We could actually get rid of the duplicate constructors too, but that's fine to save for later.